force-app/main/default/flows/Enter_Journey_Details.flow-meta.xml

```diff
@@ -0,0 +1,639 @@
+            <fieldType>InputField</fieldType>
+            <isRequired>false</isRequired>
+        </fields>
+        <fields>
+            <name>Candidate_Email</name>
+            <extensionName>flowruntime:email</extensionName>
+            <fieldType>ComponentInstance</fieldType>
+            <inputParameters>
+                <name>label</name>
+                <value>
+                    <stringValue>Candidate Email</stringValue>
+                </value>
+            </inputParameters>
+            <inputsOnNextNavToAssocScrn>UseStoredValues</inputsOnNextNavToAssocScrn>
+            <isRequired>true</isRequired>
+            <storeOutputAutomatically>true</storeOutputAutomatically>
+        </fields>
+        <fields>
+            <name>Manager_Name</name>
+            <dataType>String</dataType>
+            <fieldText>Manager Name</fieldText>
+            <fieldType>InputField</fieldType>
+            <isRequired>false</isRequired>
+        </fields>
+        <fields>
+            <name>Manager_Email</name>
+            <extensionName>flowruntime:email</extensionName>
+            <fieldType>ComponentInstance</fieldType>
+            <inputParameters>
+                <name>label</name>
+                <value>
+                    <stringValue>Manager Email</stringValue>
+                </value>
+            </inputParameters>
+            <inputsOnNextNavToAssocScrn>UseStoredValues</inputsOnNextNavToAssocScrn>
+            <isRequired>true</isRequired>
+            <storeOutputAutomatically>true</storeOutputAutomatically>
+        </fields>
+        <fields>
+            <name>Onboarding_Start_Date</name>
+            <dataType>Date</dataType>
+            <fieldText>Onboarding Start Date</fieldText>
+            <fieldType>InputField</fieldType>
+            <isRequired>false</isRequired>
+        </fields>
+        <fields>
+            <name>Edit_Template_Tasks</name>
+            <dataType>Boolean</dataType>
+            <fieldText>Edit Template Tasks?</fieldText>
+            <fieldType>InputField</fieldType>
+            <isRequired>true</isRequired>
+        </fields>
+        <showFooter>true</showFooter>
+        <showHeader>true</showHeader>
+    </screens>
+    <screens>
+        <description>Display a link allowing users to view the newly created Journey.</description>
+        <name>View_New_Journey</name>
+        <label>View New Journey</label>
+        <locationX>226</locationX>
+        <locationY>1490</locationY>
+        <allowBack>false</allowBack>
+        <allowFinish>true</allowFinish>
+        <allowPause>false</allowPause>
+        <fields>
+            <name>Display_New_Journey</name>
+            <fieldText>&lt;p&gt;&lt;a href=&quot;https://power-business-6882-dev-ed.scratch.lightning.force.com/lightning/r/Journey__c/{!Create_New_Journey}/view&quot; rel=&quot;noopener noreferrer&quot; target=&quot;_blank&quot;&gt;Journey Successfully Created &lt;/a&gt;&lt;/p&gt;</fieldText>
```

Comment:
This doesn't seem to work for me and takes me to a default login.salesforce.com page. Have you tested this? Does it work for you? 

This can be a bit a tricky one because of different environments etc, if you can paste some evidence of it working below.

Reply:
Hey Josh, please see screenshots of it working for me. Upon clicking the "Journey Successfully created" link I am taken to the Journey record.[
![Screenshot 2023-11-23 at 08 52 27](https://github.com/LAWBowie/bgAssignmentOne/assets/104722441/22d14637-fa37-4824-aac6-5b7a02a24d98)
![Screenshot 2023-11-23 at 08 52 54](https://github.com/LAWBowie/bgAssignmentOne/assets/104722441/e440b2d0-fcf4-4841-9794-86376343cb24)
](url)

Reply:
![Screenshot 2023-11-23 at 09 58 40](https://github.com/LAWBowie/bgAssignmentOne/assets/104722441/6b2304ef-5ac1-493d-a822-08c141ab83e8)